spring-apps/autosleep-core/src/main/java/org/cloudfoundry/autosleep/ui/web/model/OrgEnrollmentConfigRequest.java

```diff
@@ -0,0 +1,31 @@
+package org.cloudfoundry.autosleep.ui.web.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@Builder
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public class OrgEnrollmentConfigRequest {
+
+    @JsonProperty("idle-duration")
+    private String idleDuration;
+
+    @JsonProperty("exclude-spaces-from-auto-enrollment")
+    private String excludeSpacesFromAutoEnrollment;
+
+    @JsonProperty("state")
+    private String state;
```

Comment:
> Not necessarily. As discussed here https://www.pivotaltracker.com/n/projects/1891357/stories/134541301/comments/155991391,

I believe this discussion was related to the field ``private ServiceInstanceParameters.Enrollment autoEnrollment;`` rather than to the OrgEnrollmentConfig  field ``EnrollmentParameters.EnrollmentState state;``

> backoffice_enrolled would mean all spaces and apps inherit the values unless overridden,
backoffice_opted_out would mean all spaces and apps in that org EVEN IF enrolled separately (for example: you may have an autosleep service instance in a space of that org) would not be auto-slept.
Deleting from the org enrollment table would mean the space-level instances continue to work as is (for example: if you have a service instance in a space of that org, that continues to work).

I don't recall this discussion, but this is still a good idea and a valuable use-case.

> Do you still think we don't need the above?

Let's keep the ``state`` variable in the OrgEnrollmentConfigRequest. We might need another enum value such as to record the side effect of a DELETE (which should still survive to next org scan when the clock ticks), so we need to not delete the entry from the org enrollment table.

Here is a proposal for rewamped enum:
* ~~backoffice_enrolled~~ ``enrolled`` would mean all spaces and apps inherit the values unless overridden,
* ~~backoffice_opted_out~~ ``backoffice_recursive_opted_out``  would mean all spaces and apps in that org EVEN IF previously enrolled separately (for example: you may have an autosleep service instance in a space of that org) would not be auto-slept: all apps and spaces would be recursively unenrolled.
* ``backoffice_opted_out``: The result of a DELETE endpoint: would mean the space-level instances continue to work as is (for example: if you have a service instance in a space of that org, that continues to work).



Double checking the [current specs](https://github.com/cloudfoundry-community/autosleep/blob/develop/acceptance/org-space-enrollment-public-service-provider.feature#L216-L238), we are currently missing scenario for ``DELETE enrolled-orgs/1``. Also we would need to add another scenario for ``PATCH enrolled-orgs/1 {state=backoffice_opted_out}``. Would you like to give it a try, otherwise I'll plan to precise this over a PR